src/set_theory/game/birthday.lean

```diff
@@ -109,9 +109,6 @@ by { rw birthday_def, simp }
 @[simp] theorem birthday_star : birthday star = 1 :=
 by { rw birthday_def, simp }
 
-@[simp] theorem birthday_half : birthday half = 2 :=
-by { rw birthday_def, simpa using order.le_succ (1 : ordinal) }
-
```

Comment:
I don't see a replacement for this in this PR; I think it would be better to leave around this proof of `birthday (pow_half 1) = 2`, and remove it when you generalize in the next PR.

Reply:
This lemma is currently unused (and let's face it, not super useful). I already have #14655 more generally computing the birthday of `pow_half n`.

Reply:
Even so, PRs should be monotonic improvements; don't remove something because it's "going to be replaced later", just do the whole replacement later.

Reply:
I'm fairly happy with the rest of the PR; thanks for the detailed PR summary, it made it easy to review.

Reply:
Fair enough. I'll add this lemma back, but move it to this file.

Reply:
Thanks!